docs/releases.md

```diff
@@ -1,34 +1,37 @@
-# Release process
+# How to version and publish `frontend-shared`
 
-## 1. Version and tag locally
+## Summary
 
-We use [Semantic Versioning](https://semver.org/#semantic-versioning-200).
+1. **Update the package version** in `package.json` and merge that change into the `main` branch. We use [Semantic Versioning](https://semver.org/#semantic-versioning-200).
+2. **Create a tag** pointing at the version-change commit and generate a **new GitHub release**. Publishing a GitHub release will kick off a GitHub Action that will **publish the `@hypothesis/frontend-shared` package to `npm`.**
 
-1. Create a new working branch
-2. Run
+## 1. Update the package version
+
+1. Create a new git branch.
+2. On the new branch, run:
 
    ```shell
-   $ yarn version
+   $ yarn version --no-git-tag-version
```

Comment:
`yarn version` isn't giving us a whole lot at this point: it prompts for a new version number interactively, updates `package.json` and commits it. But it's "free" and a consistent process, so why not. The `--no-git-tag-version` option does what it sounds like it would: it suppresses the creation of a git tag.